test/binaryen.js/atomics.js.txt

```diff
@@ -1,7 +1,8 @@
+atexit() called, but EXIT_RUNTIME is not set, so atexits() will not be called. set EXIT_RUNTIME to 1 (see the FAQ)
```

Comment:
I think these messages should be deleted? (this and other files) These are only printed when the binaryen is in debug mode.

Reply:
Ah shoot, good catch.